### PR TITLE
feat(babel-preset-app): add `bugfixes` option

### PR DESCRIPTION
**Pull request by clarkdo**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Related to https://github.com/nuxt/nuxt.js/issues/6784#issuecomment-605042333

Add new option `bugfixes` which is introduced from babel-preeset-env 7.9.0.

Options can be set by:

```js
export default {
  build: {
    babel: {
      presets: ['@nuxt/babel-preset-app', { bugfixes: true }]
  }
}
```

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/babel-preset-app/src/index.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ module.exports = (api, options = {}) => {
   const envName = api.env()
 
   const {
+    bugfixes,
```
Comment (pi0) [inline packages/babel-preset-app/src/index.js]: What do you think of setting explicit default value here?

Comment (clarkdo) [reply packages/babel-preset-app/src/index.js]: I think keeping same default value as babel make it default true until babel 8 is proper.

Comment (clarkdo) [reply packages/babel-preset-app/src/index.js]: I can add false to it. And change to true when upgrade to babel 8

Comment (pi0) [reply packages/babel-preset-app/src/index.js]: Both LGTM. I let you decide state to merge :)

**Review by pi0 — APPROVED** [review]

**Pull request merged**